SimPEG/electromagnetics/static/resistivity/survey.py

```diff
@@ -203,6 +201,15 @@ def _set_abmn_locations(self):
                     locations_b.append(
                         source.location[1].reshape([1, -1]).repeat(nRx, axis=0)
                     )
+                elif isinstance(source, Src.Multipole):
```

Comment:
Hmm ... may be you are considering the case where there can be more than two current electrodes?
If then, it will be nice to add tests and a simple example. 
I think we have not done this before.

Reply:
Yeah it wouldn't be a bad idea. I've no experience writing tests though.

Reply:
Wrote the test to covert this.